pkg/netpol/eval/internal/k8s/netpol.go

```diff
@@ -144,32 +144,43 @@ func (np *NetworkPolicy) ruleConnsContain(rulePorts []netv1.NetworkPolicyPort, p
 }
 
 // ruleSelectsPeer returns true if the given peer is in the set of peers selected by rulePeers
+// also returns an indication if the rulePeers matches all namespaces in the cluster
 //
 //gocyclo:ignore
-func (np *NetworkPolicy) ruleSelectsPeer(rulePeers []netv1.NetworkPolicyPeer, peer Peer) (bool, error) {
+func (np *NetworkPolicy) ruleSelectsPeer(rulePeers []netv1.NetworkPolicyPeer, peer Peer) (peerSelected, anyNsMatch bool, err error) {
+	entireClusterCheckOnly := false
```

Comment:
the var `entireClusterCheckOnly` and its logic should be better documented

Reply:
not relevant anymore